R/R6.R

```diff
@@ -43,12 +43,16 @@ neo4j_api <- R6::R6Class(
   public = list(
     url = character(0),
     user = character(0),
+    db = character(0),
     relationships = data.frame(0),
     auth = character(0),
     labels = data.frame(0),
+    isV4 = TRUE,
```

Comment:
That's just me being picky, but I think for consistency it would be better to have `snake_case` parameters and variables everywhere